install_config/aggregate_logging.adoc

```diff
@@ -52,38 +52,24 @@ to manage container logs and prevent filling node disks.
 [[pre-deployment-configuration]]
 == Pre-deployment Configuration
 
-. Ensure that you have  deployed a router for the cluster.
-. Ensure that you have xref:../install_config/persistent_storage/index.adoc#install-config-persistent-storage-index[the
+. An Ansible playbook is available to deploy aggregated logging. You should
+familiarize yourself with the
+xref:../install_config/install/advanced_install.adoc#install-config-install-advanced-install[advanced installation and
+configuration] section. This provides information for prepairing to use Ansible
+and includes information about configuration. Parameters are added to the
+Ansible inventory file to configure various areas of the EFK stack.
+. Review the xref:../aggregate_logging_sizing.adoc#install-config-aggregate-logging-sizing[sizing guidelines]
+to determine how best to configure your deployment.
+. Ensure that you have deployed a router for the cluster.
+. Ensure that you have
+xref:../install_config/persistent_storage/index.adoc#install-config-persistent-storage-index[the
 necessary storage] for Elasticsearch. Note that each Elasticsearch replica
 requires its own storage volume. See
 xref:aggregated-elasticsearch[Elasticsearch] for more information.
-ifdef::openshift-enterprise[]
-. Ansible-based installs should create the *logging-deployer-template*
-template in the *openshift* project.
-Otherwise you can create it with the following command:
-+
-====
-----
-$ oc apply -n openshift -f \
-    /usr/share/ansible/openshift-ansible/roles/openshift_hosted_templates/files/v1.4/enterprise/logging-deployer.yaml
-----
-====
-endif::openshift-enterprise[]
-ifdef::openshift-origin[]
-. If your installation did not create templates in the *openshift* namespace, the
-*logging-deployer-template* and *logging-deployer-account-template* templates
-may not exist. In that case you can create them with the following command:
-+
-====
-----
-$ oc apply -n openshift -f \
-    https://raw.githubusercontent.com/openshift/origin-aggregated-logging/master/deployer/deployer.yaml
-----
-====
-endif::openshift-origin[]
-. Create a new project. Once implemented in a single project, the EFK stack
-collects logs for every project within your {product-title} cluster. The
-examples in this topic use *logging* as an example project:
+. Choose a project. Once deployed, the EFK stack collects logs for every
+project within your {product-title} cluster. The examples in this topic use the
+default project *logging*. The Ansible playbook creates the project for you
+if it does not already exist.
 +
 ====
 ----
```

Comment:
we do not need to describe how to create a project unless the customer wants to specify a node-selector on it. Otherwise the `openshift_logging` role will create one for us.